nix/packages/treefmt/default.nix

```diff
@@ -63,6 +63,19 @@ in
       git config --global user.name "Treefmt Test"
     '';
 
+    passthru.tests = {
+      golangci-lint = perSystem.self.treefmt.overrideAttrs (old: {
+        nativeBuildInputs = old.nativeBuildInputs ++ [pkgs.golangci-lint];
```

Comment:
There should be a better way to say: please run this derivation with all the dependencies of the other one. But I can't think of a better approach.

Reply:
Yeah, this is the simplest approach for now.